egs/wsj0-mix/DeepClustering/simple_train.py

```diff
@@ -0,0 +1,238 @@
+import os
```

Comment:
You can remove this file, if the rest of it works

Reply:
I keep it to have it in the master history but I'll remove it in the recipes's PR.